lib/Sema/CSSimplify.cpp

```diff
@@ -1415,6 +1420,15 @@ ConstraintSystem::TypeMatchResult constraints::matchCallArguments(
             cs, cs.getConstraintLocator(loc)));
       }
 
+      if (auto *param = paramInfo.getPropertyWrapperParam(argIdx)) {
+        auto argLabel = argInfo->Labels[argIdx];
+        if (cs.applyPropertyWrapperParameter(paramTy, argTy, const_cast<ParamDecl *>(param),
+                                             argLabel, subKind, locator).isFailure()) {
+          cs.getTypeMatchFailure(loc);
+        }
+        continue;
```

Comment:
Should this also check the opposite case where argument has `$` but parameter doesn't have a property wrapper? If I understand code correctly label matching just looks over `$` so if the argument type matches parameter type in this case would calls like this be wellformed?

```swift
func test(v: Int) {}

var x: Int = 42

test($v: x)
```

Reply:
Hah, I think you're right. I didn't even think about that, thank you!

Reply:
✅